core/src/it/scala/org/ensime/core/JavaCompilerSpec.scala

```diff
@@ -113,7 +113,7 @@ class JavaCompilerSpec extends EnsimeSpec
           case "3" =>
             info.name shouldBe "java.io.PrintStream.println(java.lang.Object)"
             info.localName shouldBe "println"
-            info.`type`.name shouldBe "(java.lang.Object)void"
+            info.`type`.name shouldBe "public void println(Object arg0)"
```

Comment:
now that you have the richer object it would be better to assert on the entire thing and not just the `name`... that way you can assert on the return type and named parameters.